src/components/Blog/CustomRichBlocks.tsx

```diff
@@ -102,6 +102,17 @@ export const CustomRichBlocks = ({
           );
       }
     },
+    quote: ({ children }: any) => {
```

Comment:
Would be good to type it

Reply:
Thanks! I added some more types now for the quotes + paragraph elements.